assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -54,7 +54,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Please help add below next to the 4th line.</p><p> </p><p>Copyright (c) 2020 Samuel Huang</p><p>Copyright (c) 2023 jerry-git</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>